src/ui/utils/matrixUtils.ts

```diff
@@ -21,3 +23,14 @@ export async function isRoomAliasAvailable(homeserver: string, alias: string): P
     return false;
   }
 }
+
+export function getRoomWithAlias(rooms: ObservableMap<string, Room>, alias: string): Room | void {
+  if (alias.startsWith("#") === false) return;
+  const items = rooms.values();
+
+  let item = items.next();
+  while (item.done === false) {
```

Comment:
Because `ObservableMap` implements iterable. You can use `for ... of ... ` syntax instead of this while loop. I think that syntax is a bit easier to read and less likely to cause an accidental infinite loop.